### PR TITLE
Update Line List Page to use new Ag Grid

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Breaking changes in the latest release of ag-grid require cleanup to our code.

## Related issue
N/A

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~~* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~~
~~* [ ] Tests added (or description of how to test) for any new features.~~
~~* [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

File: src/main/webapp/pages/index.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
         <h4 class="pull-left" th:text="#{event.title}">_Recent Activities_</h4>
 
         <div class="btn-group pull-right">
-          <a class="btn btn-primary btn-sm" th:href="@{/events/admin}" sec:authorize="hasRole('ROLE_ADMIN')"
+0          <a class="btn btn-primary btn-sm" th:href="@{/events/admin}" sec:authorize="hasRole('ROLE_ADMIN')"
```
Comment (deepsidhu85) [inline src/main/webapp/pages/index.html]: Remove this 0

File: src/main/webapp/resources/js/components/Tables/AgGridLayout.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import React from "react";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/components/Tables/AgGridLayout.jsx]: Add file comments

File: src/main/webapp/resources/js/pages/projects/linelist/components/InfoBar/InfoBar.jsx [hunk 3]
```diff
@@ -2,19 +2,35 @@ import React from "react";
 import PropTypes from "prop-types";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/linelist/components/InfoBar/InfoBar.jsx]: File comment

Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/linelist/components/InfoBar/InfoBar.jsx]: File comments

File: src/main/webapp/resources/js/pages/projects/linelist/components/LineList/LineListLayoutComponent.jsx [hunk 4]
```diff
@@ -4,6 +4,7 @@ import { Table } from "../Table";
 import { Toolbar } from "../Toolbar";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/linelist/components/LineList/LineListLayoutComponent.jsx]: File comment

File: src/main/webapp/resources/js/pages/projects/linelist/components/Table/renderers/IconCellRenderer.jsx [hunk 5]
```diff
@@ -16,12 +16,12 @@ export class IconCellRenderer extends React.Component {
     data: PropTypes.object.isRequired
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/linelist/components/Table/renderers/IconCellRenderer.jsx]: File comments

File: src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/Columns/ColumnVisibility.jsx [hunk 6]
```diff
@@ -2,6 +2,22 @@ import React from "react";
 import PropTypes from "prop-types";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/Columns/ColumnVisibility.jsx]: File comment

File: src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/TableControlPanel.jsx [hunk 7]
```diff
@@ -4,17 +4,48 @@ import PropTypes from "prop-types";
...
 
+import styled from "styled-components";
+import { grey2, grey5, grey9 } from "../../../../../styles/colors";
+
+const ControlPanel = styled.div`
+  height: 100%;
+  position: relative;
+  overflow: hidden;
+  background-color: ${grey2};
+  border-left: 1px solid ${grey5};
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/TableControlPanel.jsx]: For border-left should we maybe create a const in a styles/borders.js file?

File: src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/TableControlPanel.jsx [hunk 8]
```diff
@@ -4,17 +4,48 @@ import PropTypes from "prop-types";
 import { Button, Tooltip } from "antd";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/TableControlPanel.jsx]: File comments

File: src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/TemplatesPanel/TemplatesPanel.jsx [hunk 9]
```diff
@@ -2,6 +2,14 @@ import React from "react";
 import PropTypes from "prop-types";
 import { SaveTemplateModal } from "./SaveTemplateModal";
 import { TemplateSelect } from "./TemplateSelect/TemplateSelect";
+import styled from "styled-components";
+import { grey5 } from "../../../../../../styles/colors";
+
+const Wrapper = styled.div`
+  height: 75px;
+  borderbottom: 1px solid ${grey5};
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/TemplatesPanel/TemplatesPanel.jsx]: Same as comment in the above file relating to border-left

File: src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/TemplatesPanel/TemplatesPanel.jsx [hunk 10]
```diff
@@ -2,6 +2,14 @@ import React from "react";
 import PropTypes from "prop-types";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/TemplatesPanel/TemplatesPanel.jsx]: File comments

File: src/main/webapp/pages/index.html (hunk 1 above)
Comment (joshsadam) [reply src/main/webapp/pages/index.html]: Removed rogue `0` in 6912dc2504e37e03b7c8e0b2f5823519951844a4

File: src/main/webapp/resources/js/components/Tables/AgGridLayout.jsx (hunk 2 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/components/Tables/AgGridLayout.jsx]: Added in 82c9b53375c0ed3ac9654f10b56961948f76c406

File: src/main/webapp/resources/js/pages/projects/linelist/components/InfoBar/InfoBar.jsx (hunk 3 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/linelist/components/InfoBar/InfoBar.jsx]: Updated in 187bf056d0a6139ae0a9a8ff06933a64eaf374c6

File: src/main/webapp/resources/js/pages/projects/linelist/components/LineList/LineListLayoutComponent.jsx (hunk 4 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/linelist/components/LineList/LineListLayoutComponent.jsx]: Updated in 96bded4335584611075caa65245bc97e17888637

File: src/main/webapp/resources/js/pages/projects/linelist/components/Table/renderers/IconCellRenderer.jsx (hunk 5 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/linelist/components/Table/renderers/IconCellRenderer.jsx]: Updated in 987e5853c183d519a811cfe2fd0e381ee3aae5d0

File: src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/Columns/ColumnVisibility.jsx (hunk 6 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/Columns/ColumnVisibility.jsx]: Updated in 268ac92af166ed812fcb42ea5c7fc44b1466b4e2

File: src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/TableControlPanel.jsx (hunk 7 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/linelist/components/TableControlPanel/TableControlPanel.jsx]: Updated in a5a438902cda0f715ed0fb97d8d416e97eb613ec

**Comment by joshsadam** [comment]
@deepsidhu85 can you finish this one off?

**Pull request merged**